src/main/java/dev/codesoapbox/dummy4j/definitions/LocalizedDummyDefinitions.java

```diff
@@ -13,10 +13,15 @@
     String getLocale();
 
     /**
-     * Return a list of all possible values for a given key.
+     * Returns a list of all possible values for a given path.
+     * <p>
+     * Since SNAPSHOT:
```

Comment:
@since?

Reply:
In this case this is not an annotation, but rather documents some added functionality.